Cargo.toml

```diff
@@ -23,20 +23,26 @@ resolver = "2"
 [workspace.dependencies.wgc]
 package = "wgpu-core"
 git = "https://github.com/gfx-rs/wgpu"
-rev = "87576b72b37c6b78b41104eb25fc31893af94092"
-version = "0.19"
+rev = "5c5c8b1d4d2d965fbd10b290ee26f4e7eb158d7c"
+version = "22.0.0"
```

Comment:
Looks likt it pins to 22.0.0, not 22.1.0?

Reply:
Hmm, I'd used the values from wgpu's Cargo.toml:
https://github.com/gfx-rs/wgpu/blob/v22.1.0/Cargo.toml#L53-L70

But it looks like since we're using `git` dependencies, the actual version registered in `Cargo.lock` for `wgpu-core` (and friends) seems to be correct. I guess the best would be to not explicitly specify those `version` fields, and instead just let `cargo` determine them based on the git `tag`/`rev` fields.

For more info: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#the-role-of-the-version-key

Reply:
I like tagging to a tag, instead of a hash 😄